pkg/acsengine/defaults.go

```diff
@@ -98,8 +98,10 @@ func setDefaultCerts(a *api.Properties) (bool, error) {
 
 	ips := []net.IP{firstMasterIP}
 
-	for i := 1; i < a.MasterProfile.Count; i++ {
-		ips = append(ips, net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)})
+	// Include the Internal load balancer as well
+	for i := 1; i < (a.MasterProfile.Count + 1); i++ {
+		ip := net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)}
```

Comment:
Right, we're bumping up the 4th octet. Yeah, considering the way ParseIP is written: https://golang.org/src/net/ip.go?h=ParseIP#L648 (ParseIPv4 isn't exposed), net/http doesn't want to distinguish between the two at the type level i guess. We could explicitly use the IPv4, i don't feel strongly one way or the other.